index.js

```diff
@@ -280,6 +280,22 @@
     });
   };
 
+  //  EnumType :: [Any] -> Type
+  $.EnumType = function(constants) {
+    var reprs = map(constants, show);
```

Comment:
> In computer programming, an __enumerated type__ […] is a data type consisting of a set of named values called __elements__, __members__, __enumeral__, or __enumerators__ of the type.

I'd like to use one of these words in place of `constants`. I think of lists as having elements and sets as having members, so `members` gets my vote. :)